src/app/app.component.ts

```diff
@@ -132,6 +133,65 @@ export class AppComponent implements OnInit {
 
   @ViewChild('uploader') uploader: FileUploadComponent;
 
+  pillboxData = {
+    simplest: {
+      first: <PillboxContent[]>[{
+        label: 'Single label, no styling or whatever'
+      }]
+    },
+    simplestCenterdBold: {
```

Comment:
simplestCenterdBold -> simplestCenteredBold and also on line 74 of `src/app/app.component.html`?

Reply:
Thanks, I'll clean up the example code naming/duplication!

I'll ask the UX folks how they feel about this suggestion. It does seem like it'd be a simple enough option to add in the future. At the very least, if they like the idea or see a need for it, I'll add it as a code comment/TODO.